Syntaxes/Perl.plist

```diff
@@ -1366,7 +1366,7 @@
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>(?&lt;!-&gt;)\b(continue|die|do|else|elsif|exit|for|foreach|goto|if|last|next|redo|return|select|unless|until|wait|while|switch|case|require|use|eval)\b</string>
+			<string>(?&lt;!-&gt;)\b(continue|die|do|else|elsif|exit|for|foreach|goto|if|last|next|redo|return|select|unless|until|wait|while|given|when|require|use|eval)\b</string>
```

Comment:
So I presume here we started highlighting `switch|case` because it was a popular module. Looking at the documentation it is very outdated now so it is proper to remove but I wonder how common it's usage is regardless of that fact.

The three options here options here are to leave it, remove it, or leave it but mark as deprecated, not sure which is best.

@Fank Any thoughts on this?

Reply:
Anything providing switch/case as keywords is long abandoned. given/when is a slightly less ancient version of that, but has its own problems and a better replacement is in the works (keywords not finalized but will probably be match/case). given/when is still used often enough to highlight though.